lib/jekyll/post.rb

```diff
@@ -118,6 +119,13 @@ def title
       data.fetch('title') { titleized_slug }
     end
 
+    # Public: the Post excerpt_separator, from the YAML Front-Matter or empty string
+    #
+    # Returns the post excerpt_separator
+    def excerpt_separator
+      data.fetch('excerpt_separator') { "" }
```

Comment:
@parkr The fallback is above in `excerpt.rb`, though I agree that the fallback should be here instead of there.